include/klee/util/ArrayCache.h

```diff
@@ -26,15 +27,6 @@
 
 namespace klee {
 
-// FIXME: This is copied from ``klee/util/ArrayExprHash.h``.
-// We can't include it here becaues the includes are messed up
-// in that file.
-struct EquivArrayHashFn {
-  unsigned operator()(const Array *array) const {
-    return (array ? array->hash() : 0);
-  }
-};
-
 struct EquivArrayCmpFn {
```

Comment:
I could but I kept it in this header so that it's as close as possible to where it is used. Putting it in ``ArrayExprHash.h`` might suggest to some it is used by the ``ArrayExprHash`` class even though it isn't.